ios/Runner/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>20746164981-4cddb2uloqn7ustau2tag19lh7lvob8b.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.20746164981-4cddb2uloqn7ustau2tag19lh7lvob8b</string>
+	<key>API_KEY</key>
+	<string>AIzaSyBbHLJ8r33YyRNygvl5W9UdiYb-huGvdhw</string>
+	<key>GCM_SENDER_ID</key>
+	<string>20746164981</string>
+	<key>PLIST_VERSION</key>
+	<string>1</string>
+	<key>BUNDLE_ID</key>
+	<string>org.whegba.whCovid19</string>
+	<key>PROJECT_ID</key>
+	<string>whcovid19-60e64</string>
+	<key>STORAGE_BUCKET</key>
+	<string>whcovid19-60e64.appspot.com</string>
+	<key>IS_ADS_ENABLED</key>
+	<false></false>
```

Comment:
I think these are supposed to be self closing entities, ie `<false/>`

Reply:
Yep was autogenerated by FB console, so not sure. But seems to not be an issue.